db/glue.c

```diff
@@ -3841,11 +3844,14 @@ static int init_odh_llmeta(struct dbtable *d, int *compr, int *compr_blobs,
         *datacopy_odh = 0;
         return 0;
     }
-
     get_db_compress_tran(d, compr, tran);
     get_db_compress_blobs_tran(d, compr_blobs, tran);
-    get_db_instant_schema_change_tran(d, &d->instant_schema_change, tran);
-    get_db_inplace_updates_tran(d, &d->inplace_updates, tran);
+    int instant_schema_change = 0;
+    get_db_instant_schema_change_tran(d, &instant_schema_change, tran);
+    d->instant_schema_change = instant_schema_change;
+    int inplace_updates = 0;
+    get_db_inplace_updates_tran(d, &inplace_updates, tran);
+    d->inplace_updates = inplace_updates;
```

Comment:
I'm missing something. Why this change?

Reply:
i made them bitfields, and thus they can't be passed onto the functions as parameters:
/home/zaimi/comdb2/db/glue.c:3850:42: error: cannot take address of bit-field ‘instant_schema_change’
     get_db_instant_schema_change_tran(d, &d->instant_schema_change, tran);

Reply:
undoing and back to ints, code looks too ugly